ephemeris/library_data_example.yaml

```diff
@@ -0,0 +1,9 @@
+libraries:
+    - name: library_name_test_1
+      files:
+        - https://zenodo.org/record/60520/files/dbSNP_138.hg19.vcf
+    - name: library_name_test_2
+      files:
```

Comment:
@bgruening can you call this urls? Just in case we extend this to (local) files at some point.